src/styles/stars.scss

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: $width;
```

Comment:
The variable `$width` is used here, but it is not defined in this file. Ensure that it is defined in a separate SCSS file or imported correctly.